### ISSUE TITLE
Need help please

### ISSUE THREAD

**Issue by dirtyMirror** [alexyak/voicecontrol#4]
Hy guys!
First of all, I love the MagicMirror and all his awsome Modules. Also the nice and big comunity!
But I have some troubles with the voicecontroll module. I realy want to use this one. 

So here is my problem:

I want to use an USB headset for testing the voicecontroll. but i dont get it, the audio works fine if I watch movies on youtube but i think i have some troubles with the mic. Im a newbie so it would be nice if you could help me!

Tanks, for helping me!

Here is the log, when i start the Mirror:
[LOG.txt](https://github.com/alexyak/voicecontrol/files/681111/LOG.txt)

**Comment by dirtyMirror** [comment]
im using an usb headset for testing

**Comment by Gungnir83** [comment]
Hi, could you solve your Problem? I have the same now...

**Comment by dirtyMirror** [comment]
no, sorry, i dont even got an answer... maybe you try the smart mirror from
Microsoft with the pi 3

Am 14.02.2017 14:03 schrieb "Gungnir83" <notifications@github.com>:

**Comment by rcuart** [comment]
I got the same problem! >.<

**Comment by skast96** [comment]
Anyone fixed that yet ??

**Comment by kshatriypiyush** [comment]
raspberry pi can't directly recognize the voice u should buy usb microphone to make it work. the are their online specifically for raspberry
